app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1998,6 +2024,80 @@ public void run() {
         }
     }
 
+    private class ZipAlbumTask extends AsyncTask<Void, Integer, Void>{
+
+        @Override protected void onPreExecute() {
+            super.onPreExecute();
+            progressDialog.setTitle(R.string.zip_action);
```

Comment:
Notification handler class can be used to show the Zipping progress in the notification panel

Reply:
Yeah. Will do that.